composer.json

```
@@ -14,24 +14,24 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": ">=7.3.0",
+        "php": "^7.3 || ^7.4 || ^8.0 || ^8.1",
         "psr/http-message": "^1.0 || ^2.0"
     },
     "suggest": {
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
         "laminas/laminas-diactoros": "^1.8 || ^2.2",
-        "symfony/security": "^4.0 || ^5.0"
+        "symfony/security": "^4.0 || ^5.0 || ^6.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.2.1",
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
         "nyholm/psr7": "^1.0",
-        "php-coveralls/php-coveralls": "^2.2",
+        "php-coveralls/php-coveralls": "^2.2 || ^2.4",
         "phpmd/phpmd": "^2.0",
-        "phpunit/phpunit": "^8.0",
+        "phpunit/phpunit": "^8.5 || ^9.5",
```

PHPUnit 8 and 9, while technically still compatible with PHP 8.1, receive no bugfixes and are on life support: https://phpunit.de/supported-versions.html

If unit tests are passing on PHPUnit 9.5, it should be possible to support at least PHPUnit 10 with PHP 8.1:


```suggestion
        "phpunit/phpunit": "^10.0",
```